### ISSUE TITLE
Update module dependencies and fix broken build

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#118]
## Problem

We're using outdated and possibly insecure libraries. We're also missing out on fancy new features in standard-js (like the ability to auto-fix common issues). Also, [Travis seems broken](https://travis-ci.org/okTurtles/group-income-simple/jobs/163270420).
## Solution

Update deps and make sure Travis builds.

**Comment by taoeffect** [comment]
Also, feel free to remove node 5 from `.travis.yml`. E.g. change:

```yml
node_js:
  - "5"
  - "6"
```

To:

```yml
node_js:
  - "6"
```

**Comment by taoeffect** [comment]
OK, note that I did the above in PR #126.

I also attempted to fix travis [by fixing the issues `hapi.js` and `interface.js`](https://github.com/taoeffect/group-income-simple/commit/43a675e36ae2dab3fed731ea089b38522e880f1b) (also part of that PR), but it seems [there's another issue](https://travis-ci.org/okTurtles/group-income-simple/builds/179330222) with the build where `Should create user George` fails. I'll leave that to your capable hands.

Note: since I touched the same files (hapi.js and interface.js), I recommend either making your files match mine identically to avoid a merge conflict, or, after merging #126, rebasing your work on top of it.
